pkg/operator/ceph/cluster/rbd/spec.go

```diff
@@ -75,7 +75,7 @@ func (r *ReconcileCephRBDMirror) makeDeployment(daemonConfig *daemonConfig, rbdM
 			Name:        daemonConfig.ResourceName,
 			Namespace:   rbdMirror.Namespace,
 			Annotations: rbdMirror.Spec.Annotations,
-			Labels:      controller.CephDaemonAppLabels(AppName, rbdMirror.Namespace, config.RbdMirrorType, daemonConfig.DaemonID, true),
+			Labels:      controller.CephDaemonAppLabels(AppName, rbdMirror.Namespace, config.RbdMirrorType, daemonConfig.DaemonID, daemonConfig.ResourceName, "CephrbdMirror", true),
```

Comment:
Is the CRD name expected here (cephrbdmirrors.ceph.rook.io) instead of the short name (CephrbdMirror)? And if we expect the short name, CephRBDMirror is more consistent with caps preserved